enderio-conduits/src/main/java/com/enderio/conduits/client/model/conduit/facades/FacadeChunkReloadHelper.java

```diff
@@ -0,0 +1,36 @@
+package com.enderio.conduits.client.model.conduit.facades;
+
+import com.enderio.base.common.item.tool.YetaWrenchItem;
+import com.enderio.conduits.EnderIOConduits;
+import net.minecraft.client.Minecraft;
+import net.minecraft.world.entity.EquipmentSlot;
+import net.minecraft.world.entity.player.Player;
+import net.neoforged.api.distmarker.Dist;
+import net.neoforged.bus.api.SubscribeEvent;
+import net.neoforged.fml.common.EventBusSubscriber;
+import net.neoforged.neoforge.event.entity.living.LivingEquipmentChangeEvent;
+
+@EventBusSubscriber(modid = EnderIOConduits.MODULE_MOD_ID, bus = EventBusSubscriber.Bus.GAME, value = Dist.CLIENT)
+public class FacadeChunkReloadHelper {
+    @SubscribeEvent
+    public static void onEquipmentChanged(LivingEquipmentChangeEvent event) {
+        if (event.getSlot() != EquipmentSlot.MAINHAND &&
+            event.getSlot() != EquipmentSlot.OFFHAND) {
+            return;
```

Comment:
`|| (!event.getEntity() instanceof LocalPlayer)` is a shorter check than what happens next.

Reply:
Good call, thanks!